builtin/rebase.c

```diff
@@ -582,7 +603,6 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 	const char *gpg_sign = NULL;
 	int ret, flags, total_argc, in_progress = 0;
 	int ok_to_skip_pre_rebase = 0;
-	int autostash = 0;
```

Comment:
I guess you can leave it at that, as the original commit did not need `autostash` in the `rebase_options`. But the commit message needs to prepare the reviewer that that field was introduced for the sole purpose of having it available for handling the config setting.